### PR TITLE
[test](multi-catalog)Regression test for external hive parquet table.

### PR DESCRIPTION
**Pull request by Jibing-Li**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Regression test for external hive parquet table. Using docker to provide hive environment. 

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/external_catalog_p0/hive/test_hive_parquet.groovy [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    String enabled = context.config.otherConfigs.get("enableHiveTest")
+    if (enabled != null && enabled.equalsIgnoreCase("true")) {
+        try {
+            String hms_port = context.config.otherConfigs.get("hms_port")
+            sql """admin set frontend config ("enable_multi_catalog" = "true")"""
+            sql """admin set frontend config ("enable_new_load_scan_node" = "true");"""
+            set_be_config.call('true')
+            sql """drop catalog if exists hive"""
+            sql """
+            create catalog hive properties (
```
Comment (morningman) [inline regression-test/suites/external_catalog_p0/hive/test_hive_parquet.groovy:154]:
```suggestion
            create catalog if not exists hive properties (
```

File: regression-test/suites/external_catalog_p0/hive/test_hive_parquet.groovy [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                "type"="hms",
+                'hive.metastore.uris' = 'thrift://127.0.0.1:${hms_port}'
+            );
+            """
+            sql """switch hive"""
+            sql """use `default`"""
```
Comment (morningman) [inline regression-test/suites/external_catalog_p0/hive/test_hive_parquet.groovy:160]:
```suggestion
            sql """use hive.default"""
```

File: regression-test/suites/external_catalog_p0/hive/test_hive_parquet.groovy [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+            sql """admin set frontend config ("enable_new_load_scan_node" = "true");"""
+            set_be_config.call('true')
+            sql """drop catalog if exists hive"""
+            sql """
+            create catalog hive properties (
+                "type"="hms",
+                'hive.metastore.uris' = 'thrift://127.0.0.1:${hms_port}'
+            );
+            """
+            sql """switch hive"""
```
Comment (morningman) [inline regression-test/suites/external_catalog_p0/hive/test_hive_parquet.groovy:159]:
```suggestion
```

File: regression-test/suites/external_catalog_p0/hive/test_hive_parquet.groovy (hunk 2 above)
Comment (morningman) [reply regression-test/suites/external_catalog_p0/hive/test_hive_parquet.groovy:160]: We can test this `use` case by the way.

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.77 seconds
 load time: 441 seconds
 storage size: 17153249327 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221114043412_clickbench_pr_45373.html

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**